Mathlib/Logic/Small/Defs.lean

```diff
@@ -93,6 +93,9 @@ instance small_ulift (α : Type u) [Small.{v} α] : Small.{v} (ULift.{w} α) :=
 theorem small_type : Small.{max (u + 1) v} (Type u) :=
   small_max.{max (u + 1) v} _
 
+instance {α : Type u} [Small.{v} α] [Nontrivial α] : Nontrivial (Shrink.{v} α) :=
+  (equivShrink α).symm.nontrivial
+
```

Comment:
Was this deliberate? It seems unrelated to the rest of the PR.

Reply:
Please note that in the PR description before merging (or split out a separate PR).